meilisearch/tests/index/test_index.py

```diff
@@ -140,6 +140,8 @@ def test_update_index(self):
         index = self.client.index(uid=self.index_uid)
         response = index.update(primaryKey='objectID')
         assert isinstance(response, object)
```

Comment:
```suggestion
        assert isinstance(response, Index)
```

Reply:
I wanted to, but impossible, the `meilisearch.Index` is not accessible during import, or maybe I'm wrong.